### DISCUSSION TITLE
Support for Pascal-based Quadros?

### DISCUSSION THREAD

**Discussion by SamantazFox** [NVIDIA/open-gpu-kernel-modules#611] [Q&A] [answered]
Are there any plans to add support for the Pascal based GPUs? I have a fairly recent Quadro P2200 (from mid 2020) and I'd really like to see it supported by the these drivers.

**Comment by ttabi** [comment]
Pascal GPUs lack the hardware necessary.  This has been discussed ad nauseum.

**Comment by mtijanic** [comment, marked as answer]
Everything about this should be fully explained in the pinned #19 issue and the several discussion threads that forked from there. If you have questions that haven't been answered, feel free to post there, but there's no need for another thread, so closing.

**Reply by SamantazFox** [reply]
Oh, I didn't see the pinned issue, sorry D:
